app/presenters/users/dashboard_presenter.rb

```diff
@@ -49,36 +49,31 @@ def courses
             )
           else
             current_school.courses.live.where(
-              id:
-                (
-                  courses_with_student_profile.pluck(:course_id) +
-                    courses_with_review_access + courses_with_author_access
-                ).uniq
+              id: (courses_with_student_profile.pluck(:course_id) +
+                   courses_with_review_access + courses_with_author_access).uniq
             )
           end.with_attached_thumbnail
         end
     end
 
     def courses_with_student_profile
-      @courses_with_student_profile ||=
-        begin
+      @courses_with_student_profile ||= begin
           current_user
             .founders
             .joins(:course)
             .pluck(:course_id, :dropped_out_at, :access_ends_at)
             .map do |course_id, dropped_out_at, access_ends_at|
-              {
-                course_id: course_id,
-                dropped_out_at: dropped_out_at,
-                access_ends_at: access_ends_at
-              }
-            end
+            {
+              course_id: course_id,
+              dropped_out_at: dropped_out_at,
+              access_ends_at: access_ends_at
+            }
+          end
```

Comment:
Incorrect formatting, please use the [recommended config](https://docs.pupilfirst.com/developers/development_setup#code-formatting)